frontend/src/App.js

```diff
@@ -65,10 +66,11 @@ function App() {
           <Route path="/pdf" element={(currentUser && currentUser.employee) ? <InvoicePreview /> : (currentUser ? <Setting /> : <LoginForm from={window.location.pathname} />)} />
           <Route path="/login" element={<LoginForm />} />
           <Route path="/signup" element={<SignUpForm />} />
+          {/* <Route path="/forgotpassword" element={<ForgotPasswordForm />} /> */}
```

Comment:
We can uncomment all these and add it in, if there's no entry point, it won't be accessible anyways so no need to comment